### PR TITLE
dolt-gc: add page

### PR DESCRIPTION
**Pull request by styffnn**
- [x] The page(s) are in the correct platform directories: common, linux, osx, windows, sunos, android, etc.
- [x]  The page(s) have at most 8 examples.
- [x]  The page description(s) have links to documentation or a homepage.
- [x]  The page(s) follow the [content guidelines](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x]  The PR title conforms to the recommended [templates](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/dolt-gc.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿# dolt gc
+
+> Search the repository for data that is no longer referenced and no longer needed.
```
Comment (gutjuri) [inline pages/common/dolt-gc.md:3]:
```suggestion
> Search the repository for data that are no longer referenced and no longer needed.
```

"Data" is plural

**Review by gutjuri — COMMENTED** [review]
Thank you! I have one suggestion.

**Comment by styffnn** [comment]
Could someone assist me? I'm uncertain about how to resolve this error.

**Comment by fazlearefin** [comment]
> Could someone assist me? I'm uncertain about how to resolve this error.
> 
> `pages/common/dolt-gc.md:1: TLDR001 File should contain no leading whitespace`

Your file has some invisible character at the beginning of the file. I fixed it using `dos2unix` utility. You need to something similar in your workstation and push the commit again. See image below.

Screenshot from 2024-03-06 04-49-46

**Comment by styffnn** [comment]
> > Could someone assist me? I'm uncertain about how to resolve this error.
> > `pages/common/dolt-gc.md:1: TLDR001 File should contain no leading whitespace`
> 
> Your file has some invisible character at the beginning of the file. I fixed it using `dos2unix` utility. You need to something similar in your workstation and push the commit again. See image below.
> 
> Screenshot from 2024-03-06 04-49-46

Thank you for your help, I really appreciate it!

**Review by gutjuri — APPROVED** [review]
Thanks for implementing my feedback! LGTM!

**Review by fazlearefin — APPROVED** [review]

**Pull request merged**